content/batch/install.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Install Argo CLI"
+date: 2018-11-18T00:00:00-05:00
+weight: 30
+draft: false
+---
+
+### Install Argo CLI
+
+Before we can get started configuring `argo` we'll need to first install the
+command line tools that you will interact with. To do this run the following.
+
+```
+curl -sSL -o /usr/local/bin/argo https://github.com/argoproj/argo/releases/download/v2.2.1/argo-linux-amd64
+chmod +x /usr/local/bin/argo
+```
```

Comment:
above commands failed but adding sudo worked for me

dalbhanj:~/environment $ curl -sSL -o /usr/local/bin/argo https://github.com/argoproj/argo/releases/download/v2.2.1/argo-linux-amd64
curl: (23) Failed writing body (0 != 16360)
dalbhanj:~/environment $ sudo curl -sSL -o /usr/local/bin/argo https://github.com/argoproj/argo/releases/download/v2.2.1/argo-linux-amd64
dalbhanj:~/environment $ chmod +x /usr/local/bin/argo
chmod: changing permissions of ‘/usr/local/bin/argo’: Operation not permitted
dalbhanj:~/environment $ sudo chmod +x /usr/local/bin/argo

Reply:
Fixed in future commit.